src/nvim/buffer_defs.h

```diff
@@ -1373,4 +1374,7 @@ struct window_S {
   size_t w_winbar_click_defs_size;              // Size of the w_winbar_click_defs array
   StlClickDefinition *w_statuscol_click_defs;   // Status column click definitions
   size_t w_statuscol_click_defs_size;           // Size of the w_statuscol_click_defs array
+                                                //
+  bool w_want_scrollbar;                // when false don't use a scrollbar
```

Comment:
since we want any window to be able to have a scrollbar, add these two fields, used later to determine whether a window has a scrollbar and whether it should be displayed.